.github/workflows/publish-all.yml

```diff
@@ -12,7 +12,8 @@ on:
       - staging
       - nf/publish-apps
       - main
-      # TODO: enable production and remove publish.yml
+  release:
+    types: [published]
```

Comment:
ooo one gotcha here is that I'm not sure the `branch` here is going to be set to `production`... release please runs on the `rc` branch and so the release gets tagged there. That commit then gets pushed to `production` but not until the next step.

The main reason we need to run on this trigger is to be able to upload things to the release so perhaps what we should do is:
- also include `production` in the branches list
- publish apps and npm packages from production but not release trigger
- upload artifacts from release trigger but not production
- in retrospect `publish_devtools_extension` is probably useless because I'm pretty sure it will only work if triggered from a release tag

Reply:
What's the reason for uploading artifacts from the release trigger?

I had initially thought there'd be a downside in having to build twice, but now I'm thinking we should probably enable it for rc branches too?

Reply:
> What's the reason for uploading artifacts from the release trigger?

it's been quite a while since I first set this up, but iirc the action we're using for upload only works if run from the release trigger. Maybe that's how it associates the uploads to the correct release?

Reply:
It looks like it bases the version on the version tag. so if we added the rc branches, I think it'd just work.

Reply:
@nferch wouldn't that trigger publish on every push to the `rc` branches though?

Reply:
Yeah, but it shouldn't do any harm and is kinda what we want anyway? We'll know whether any of the commits to the rc branch fail to build or publish.

Reply:
for rc (and hotfix) I think we'd want to bundle & build but not publish apps/npm? but then do attempt to upload extension artifacts?